src/main/java/group1/mips_simulator/components/RomLoader.java

```diff
@@ -0,0 +1,8 @@
+package group1.mips_simulator.components;
+
+public class RomLoader {
+    /*
+    read the boot program from the ROM and place it into memory in a location you designated.
```

Comment:
Any idea on where in memory we should put the program? I'll suggest exactly at the half way point in our memory. So like $0 -> $half is for storage and $(half + 1) -> $end  is for program instructions. But that's just a suggestion I'm open to other ideas.

Reply:
He's suggested a location in the project description:
Under 3.1 Central Processor
...
NOTE: Thus, the first address of your program should be greater than your length of the program.
I suggest you load your boot program beginning at location octal 10 and the first address of your program at octal address = octal 10 + length of boot program + octal 10
...

I'm still deciphering what exactly this means. I dont know if its helpful, but im aware that the standard memory management architecture is usually something like:
[heap]. //dynamic variables
[.  v   ]
[.  -   ]
[.  ^  ]
[stack] //paramaters
[static] //global/static data
[code] //instructions

I dont know how true to this we need to stick for the project, so lemme think on that
tbc not anti the suggestion, just not sure what profs looking for